content/releases/cloudnative-pg-1-24.0-rc1-released/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "CloudNativePG 1.24.0 RC1 Released!"
+date: 2024-07-26T15:17:37+02:00
+draft: false
+author: gbartolini
+image:
+    url: 58364dde1adc4a0a8.37186404-2048x1445.jpg
+    attribution: from <a href="https://wordpress.org/photos/photo/58364dde1a/">Saurabh</a>
+tags:
+ - release
+ - postgresql
+ - postgres
+ - kubernetes
+ - k8s
+ - cloudnativepg
+summary: "The CloudNativePG community has released the first release candidate of CloudNativePG 1.24. Help us test new features in preview like distributed PostgreSQL topologies, managed services, enhanced API for synchronous replication, and WAL disk space safeguards."
+---
+The **CloudNativePG Community** is thrilled to announce the release of the
+first candidate for CloudNativePG 1.24! This release candidate showcases all
+the features expected in the final version of CloudNativePG 1.24, although some
+details may still be refined during this preview phase.
+
+For detailed information on the features and changes in CloudNativePG 1.24,
+please refer to the [release notes](https://cloudnative-pg.io/documentation/preview/release_notes/v1.24/).
+
+### Get Involved and Make a Difference!
+
+In true open source spirit, we strongly encourage you to test the new features
+of CloudNativePG 1.24 on your systems. Your testing and feedback are crucial in
+helping us identify and resolve any bugs or issues. While we do not recommend
+using CloudNativePG 1.24 RC1 in production environments, we urge you to
+simulate your typical application workloads with this preview release.
+
+Your involvement ensures that the final release of CloudNativePG 1.24 upholds
+our high standards of stability and reliability, maintaining its status as one
+of the world's most popular open source operators for PostgreSQL in Kubernetes.
+
+For more details on our preview testing process and how you can contribute,
+read more [here](https://cloudnative-pg.io/documentation/current/preview_version).
+
+---
+
+## CloudNativePG 1.24 Feature Highlights
+
+### Distributed PostgreSQL Topologies
+
+CloudNativePG 1.24 significantly enhances the replica cluster feature, enabling
+the creation of distributed database topologies for PostgreSQL across multiple
+Kubernetes clusters. This supports hybrid and multi-cloud deployments,
+offering:
+
+- **Declarative Primary Control**: Use declarative configuration to easily
+  specify which PostgreSQL cluster acts as the primary in a distributed setup.
+- **Seamless Switchover**: Demote the current primary and promote a selected
+  replica cluster, typically in a different region, without needing to rebuild
+  the former primary. This ensures high availability and resilience in diverse
+  environments.
```

Comment:
"environments" sounds here a bit like "availability zones" or clusters. I think we could talk about use cases or scenarios.
```suggestion
  use cases.
```

Reply:
Maybe 'scenarios'?